Dockerfile

```diff
@@ -15,14 +15,19 @@ COPY api/ api/
 COPY pkg/ pkg/
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
+ARG TARGETARCH
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=${TARGETARCH} GO111MODULE=on go build -a -o manager-${TARGETARCH} main.go
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-# oamdev/gcr.io-distroless-static:nonroot is syncd from gcr.io/distroless/static:nonroot as somewhere can't reach gcr.io
-FROM oamdev/gcr.io-distroless-static:nonroot
+# Could use `--build-arg=BASE_DISTROLESS=gcr.io/distroless/static:nonroot` to overwrite
+ARG BASE_DISTROLESS
+FROM ${BASE_DISTROLESS:-gcr.io/distroless/static:nonroot}
```

Comment:
official `gcr.io/distroless/static:nonroot` support multi-arch now.
for local build. if can't reach `gcr.io`. 
`--build-arg=BASE_DISTROLESS` to overwrite

could use a fork https://github.com/querycap/distroless.

Reply:
Yes, `gcr.io` isn't an option due to network connection issues.

Reply:
@morlay  excellent solution！